src/ui/builder.js

```diff
@@ -189,6 +189,13 @@ cwc.ui.Builder = function() {
   }
 };
 
+/**
+ * @return {!cwc.utils.Helper}
+ * @export
+ */
+cwc.ui.Builder.prototype.getHelper = function() {
+  return this.helper;
```

Comment:
It's used by test/core_tests/test_tutorial.js, which can't access 'helper' because the compiler renames it. Adding @export to the 'helper' property generates an compiler error. I think giving the compiler the --export_local_property_definitions flag would fix that, but I'm unsure what the wider ramifications of that flag would be.